### ISSUE TITLE
RUN: All chained commands should be parsed

### ISSUE THREAD

**Issue by fhemberger** [replicatedhq/dockerfilelint#110]
```
RUN apk update \
  && apk add ca-certificates \ 
  && rm -rf /var/cache/apk/*
```

=> "Consider `--no-cache` or `--update` with `rm -rf /var/cache/apk/*` (Optimization)"

The parser should detect the `rm -rf …` in the third line.

```
RUN apk update \
  && rm -rf /var/cache/apk/*
```

=> No error message.
